### PR TITLE
docs: add change log for 2.1.0

### PR DESCRIPTION
**Pull request by benjycui**

### CODE REVIEW

File: CHANGELOG.zh-CN.md [hunk 1]
```diff
@@ -9,6 +9,37 @@ timeline: true
 
 ---
 
+## 2.1.0
+
+`2016-10-14`
+
+- Icon 现在支持旋转动画。
+- Tabs 现在可以禁用切换动画。
+- 新增西班牙语的 l10n 支持。@Danjavia
```
Comment (afc163) [inline CHANGELOG.zh-CN.md]: i10n?

File: CHANGELOG.zh-CN.md [hunk 2]
```diff
@@ -9,6 +9,37 @@ timeline: true
...
+- 更新俄语的 l10n 文案。@plandem
+- 新增 AutoComplete[onSelect] 回调。
+- 优化 Modal 样式细节。
+- 优化 Tooltip 动画。
+- 优化 Transfer 按钮的样式。
+- 修复 Affix 与 BackTop 的服务端渲染问题。[#3283](https://github.com/ant-design/ant-design/issues/3283) [#3343](https://github.com/ant-design/ant-design/issues/3343)
+- 修复 Cascader 搜索模式与浏览器自动完成有冲突的问题。[#3350](https://github.com/ant-design/ant-design/issues/3350)
+- 修复 DatePicker 与 TimePicker 样式冲突问题。[#3312](https://github.com/ant-design/ant-design/issues/3312) [#3307](https://github.com/ant-design/ant-design/issues/3307)
```
Comment (afc163) [inline CHANGELOG.zh-CN.md]: 合并到下面吧。

**Comment by benjycui** [comment]
Done.

**Comment by afc163** [comment]
https://github.com/ant-design/ant-design/pull/3416
https://github.com/ant-design/ant-design/pull/3399
https://github.com/ant-design/ant-design/issues/3363

这几个问题补充和确认掉应该就够了。

**Pull request merged**